lnrpc/routerrpc/forward_interceptor.go

```diff
@@ -119,6 +121,24 @@ func (r *forwardInterceptor) resolveFromClient(
 			Action: htlcswitch.FwdActionResume,
 		})
 
+	case ResolveHoldForwardAction_RESUME_MODIFIED:
+		// Modify HTLC and resume forward.
+		outgoingAmtMsat := lnwire.MilliSatoshi(in.OutgoingAmountMsat)
```

Comment:
So do we expect the RPC clients to set this value based off the fee policy? Consider that if we're traversing a channel that has fees, they can't just set this to the incoming value. The outgoing link would reject as not enough fees have been paid.

Reply:
So maybe we have this be `None` if the value from the RPC response is `0`?

Reply:
Gone with `None` if RPC response is 0.